### ISSUE TITLE
v0.2 binary release won't start on M2 Macbook Pro

### ISSUE THREAD

**Issue by tveastman** [beeware/podium#60]
### Describe the bug

Hi! I'm not sure I have the no-how to roll my own release. The current release v0.2 doesn't seem to do anything when I try to run it from my work laptop.

(No error, no visible evidence that it ran at all)

### Steps to reproduce

1. Downloaded the 0.2 release and put it in "Applications".
2. I'm pretty sure did the command-open thingee you often have to do when running unsigned releases?
3. I've tried running podium by (a) launching the app, (b) using `open Podium.app` in the terminal, and opening a `.podium` presentation.
4. In all cases, nothing happens, not even an error message.

### Expected behavior

Just expecting the app to launch, although a decipherable error would be nice too.

### Environment

Trying to use the build release, so I don't know if python versions etc are relevant.

Apple M2 Pro, macOS Ventura 13.5

**Comment by freakboy3742** [comment]
In terms of know how, the process should be:
* `pip install briefcase`
* `briefcase package --adhoc`

which should generate a DMG that contains an adhoc signed binary that will work on your machine and nobody else's. 

However, I should definitely publish an updated binary that is fully signed, and includes all the recent bug fixes and tweaks from recent Toga versions.

**Comment by tveastman** [comment]
You're right, that turned out to be absurdly easy to build my own v.03 that works! Awesome!
